### PR TITLE
Feature/get version from build info

### PR DESCRIPTION
**Pull request by jbern0rd**

### CODE REVIEW

File: src/main/java/com/iexec/resultproxy/version/VersionService.java [hunk 1]
```diff
@@ -2,21 +2,22 @@
...
+import org.springframework.boot.info.BuildProperties;
 import org.springframework.stereotype.Service;
 
 @Service
 public class VersionService {
 
-    @Value("${version}")
-    private String version;
+    @Autowired
+    BuildProperties buildProperties;
```
Comment (jeremyjams) [inline src/main/java/com/iexec/resultproxy/version/VersionService.java:13]: private field + injection in constructor?

**Review by mcornaton — APPROVED** [review]
Really clean, thanks!

File: src/main/java/com/iexec/resultproxy/version/VersionService.java (hunk 1 above)
Comment (jbern0rd) [reply src/main/java/com/iexec/resultproxy/version/VersionService.java]: see https://github.com/iExecBlockchainComputing/iexec-result-proxy/pull/45/commits/2118178c58a92d13b24f8a4c917f3d078dc375d7

**Review by jeremyjams — APPROVED** [review]
Nice and clean!

**Pull request merged**